### ISSUE TITLE
移除 gnome

### ISSUE THREAD

**Issue by taoky** [ustclug/mirrorrequest#401]
### 先决条件 (Prerequisites)

- [X] 我已确认该事宜从未在 [其他 issues](https://github.com/ustclug/mirrorrequest/issues) 中讨论过。  
I have checked that my proposal has not been discussed in [other issues](https://github.com/ustclug/mirrorrequest/issues).

- [X] 我已仔细阅读 [请求新镜像前必读](https://github.com/ustclug/mirrorrequest/issues/350)。  
I have CAREFULLY read the [mirroring rules](https://github.com/ustclug/mirrorrequest/issues/350).

### 修改或删除的原因 (Reasons of the modification or removal)

- https://gitlab.gnome.org/Infrastructure/Infrastructure/-/issues/1796
- https://discourse.gnome.org/t/gnome-mirroring-system-updates/25031

tl;dr: GNOME 上游转向 CDN77，不再提供 rsync 服务、维护镜像列表。

### 配置方法 (Configuration method)

N/A

### 修改后的镜像大小 (Mirror size after modification)

-316.95 GB

**Comment by taoky** [comment]
Removed.
